### PR TITLE
Fix osx build

### PR DESCRIPTION
**Pull request by warting**
Fix osx build

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -66,7 +66,6 @@ kotlin {
                 implementation 'com.fasterxml.jackson.core:jackson-databind:2.9.8'
                 implementation 'com.fasterxml.jackson.module:jackson-module-kotlin:2.9.8'
                 implementation 'com.fasterxml.jackson.module:jackson-module-parameter-names:2.9.8'
-                implementation "org.jetbrains.kotlinx:kotlinx-io-jvm:$kotlinx_io_version"
```
Comment (thomasletsch) [inline build.gradle]: Shouldn't that line remain? We would get problems under non-MacOS platforms, no?

File: build.gradle [hunk 2]
```diff
@@ -79,7 +78,6 @@ kotlin {
         jsMain {
             dependencies {
                 implementation 'org.jetbrains.kotlin:kotlin-stdlib-js'
-                implementation "org.jetbrains.kotlinx:kotlinx-io-js:$kotlinx_io_version"
```
Comment (thomasletsch) [inline build.gradle]: Same here, removing the dependency to kotlinx-io will break the JS build, no?

**Review by thomasletsch — COMMENTED** [review]
Hi Stefan,
don't know if we break other things by fixing it this way :-)

File: build.gradle (hunk 1 above)
Comment (warting) [reply build.gradle]: yes it should remain. I'll put it back

**Review by thomasletsch — APPROVED** [review]

**Pull request merged**